Translations/pt_BR.po

```diff
@@ -794,443 +794,437 @@ msgid "Read-Only Directory Error"
 msgstr "Erro de diretório, somente leitura"
 
 #: Source/automap.cpp:434 Source/items.cpp:3864
-#, fuzzy, c-format
-#| msgid "Level:"
+#, c-format
 msgid "Level: %i"
-msgstr "Nível:"
+msgstr "Nível: %i"
 
 #: Source/automap.cpp:436
 #, c-format
 msgid "Level: Crypt %i"
-msgstr ""
+msgstr "Nível: Cripta %i"
 
 #: Source/automap.cpp:438
-#, fuzzy, c-format
-#| msgid "Level:"
+#, c-format
 msgid "Level: Nest %i"
-msgstr "Nível:"
+msgstr "Nível: Ninho %i"
 
 #: Source/control.cpp:244
 msgid "Character Information"
-msgstr ""
+msgstr "Informação do personagem"
 
 #: Source/control.cpp:245
 msgid "Quests log"
-msgstr ""
+msgstr "Registro de missões"
 
 #: Source/control.cpp:246
 msgid "Automap"
-msgstr ""
+msgstr "Automapa"
 
 #: Source/control.cpp:247
 msgid "Main Menu"
-msgstr ""
+msgstr "Menu principal"
 
 #: Source/control.cpp:248
 msgid "Inventory"
-msgstr ""
+msgstr "Inventário"
 
 #: Source/control.cpp:249
 msgid "Spell book"
-msgstr ""
+msgstr "Livro de feitiços"
 
 #: Source/control.cpp:250
 msgid "Send Message"
-msgstr ""
+msgstr "Enviar mensagem"
 
 #: Source/control.cpp:251
 msgid "Player Attack"
-msgstr ""
+msgstr "Hostil a jogadores"
 
 #: Source/control.cpp:435 Source/control.cpp:1029
 #, c-format
 msgid "%s Skill"
-msgstr ""
+msgstr "Habilidade de %s"
 
 #: Source/control.cpp:438 Source/control.cpp:1033
 #, c-format
 msgid "%s Spell"
-msgstr ""
+msgstr "Feitiço de %s"
 
 #: Source/control.cpp:440
 #, c-format
 msgid "Damages undead only"
-msgstr ""
+msgstr "Danos apenas em mortos-vivos"
 
 #: Source/control.cpp:444 Source/control.cpp:1039 Source/control.cpp:1865
 #, c-format
 msgid "Spell Level 0 - Unusable"
-msgstr ""
+msgstr "Nível do feitiço 0 - Inusável"
```

Comment:
"Inusável" doesn't exist in Portuguese. "Inutilizável" is the correct word.

Wouldn't it be better to use "Magia" instead of "Feitiço"?